### PR TITLE
Polar legends

### PR DESCRIPTION
**Pull request by danmarshall**

### CODE REVIEW

File: src/app/views/panels/scale_value_selector.tsx [hunk 1]
```diff
@@ -136,7 +136,8 @@ export class ScaleValueSelector extends React.Component<
                   onClick={() => {
                     new Actions.ToggleLegendForScale(
                       scale._id,
-                      scaleMapping
+                      scaleMapping,
+                      null // TODO find the plot segment
```
Comment (danmarshall) [inline src/app/views/panels/scale_value_selector.tsx:140]: Fix this

File: src/core/prototypes/plot_segments/region_2d/polar.ts [hunk 2]
```diff
@@ -333,12 +457,39 @@ export class PolarPlotSegment extends PlotSegmentClass<
...
+    } = attrs;
     return [
       { type: "x", value: x1, attribute: "x1" } as SnappingGuides.Axis,
       { type: "x", value: x2, attribute: "x2" } as SnappingGuides.Axis,
       { type: "y", value: y1, attribute: "y1" } as SnappingGuides.Axis,
       { type: "y", value: y2, attribute: "y2" } as SnappingGuides.Axis,
+      { type: "x", value: cx, attribute: "cx" } as SnappingGuides.Axis,
+      { type: "y", value: cy, attribute: "cy" } as SnappingGuides.Axis,
+
+      // do we need snapping to these ??????????
```
Comment (danmarshall) [inline src/core/prototypes/plot_segments/region_2d/polar.ts:484]: No

File: src/app/views/panels/scale_value_selector.tsx (hunk 1 above)
Comment (danmarshall) [reply src/app/views/panels/scale_value_selector.tsx:140]: @ilfat, I may have [a fix for this](https://github.com/microsoft/charticulator/commit/8c6fe1a77babf53bd83e04b23ba251640aca90b3), but I am unsure of how this code is invoked.

Comment (zBritva) [reply src/app/views/panels/scale_value_selector.tsx:140]: Add checks PS for null in toggleLegendForScale and pass null here

**Review by zBritva — APPROVED** [review]

**Review by bongshin — APPROVED** [review]

**Pull request merged**